analyzers/src/SonarAnalyzer.Common/SonarAnalyzer.Common.csproj

```diff
@@ -21,6 +21,10 @@
       <!-- Downgrade System.Collections.Immutable to support VS2015 Update 3 -->
       <NoWarn>NU1605, NU1701</NoWarn>
     </PackageReference>
+    <!--  Manually upgrade since the version transitively linked (4.0.1) is marked as vulnerable by WhiteSource. -->
+    <PackageReference Include="System.IO.FileSystem" Version="4.3.0" />
```

Comment:
This `FileSystem` doesn't seem intentional to me.
```suggestion
```

Reply:
Can we move this `System.IO.FileSystem` thing to Rule descriptor, as it fails only there? I gave it a try and the build worked.

We'd avoid messing with production project to please a build tooling.
In the long run, I want to get rid of the rule descriptor one day so the workaround would vanish with it.